api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -2130,7 +2130,7 @@ def detect_liquid_presence(self, well: labware.Well) -> bool:
 
     @requires_version(2, 20)
     def require_liquid_presence(self, well: labware.Well) -> None:
-        """If there is no liquid in a well, raise an error.
+        """Checks for liquid in a well. Raises an error if no liquid is detected. Flex pipettes only. See :ref:`lpd` and :ref:`require-liquid-presence`.
```

Comment:
```suggestion
        """Checks for liquid in a well and raises an error if none is detected. 
        
        When this method raises an error, Flex will offer the opportunity to enter recovery mode. In recovery mode, you can manually add liquid to resolve the error. 
        
        Works with Flex pipettes only. See :ref:`lpd` and :ref:`require-liquid-presence`.
```

Suggesting to differentiate the first lines and explain a bit more about what you can do when an error happens. (For power users, this also implies that you don't need to catch this error in a `try...except` block.)